recipes/stlab/all/conanfile.py

```diff
@@ -1,180 +1,179 @@
         else:
-            msg += "Try -o boost_variant=True."
-
-        raise ConanInvalidConfiguration(msg)
+            minimum_version = self._compilers_minimum_version.get(str(self.settings.compiler), False)
+            if not minimum_version:
+                self.output.warn(f"{self.ref} requires C++{self._minimum_cpp_standard}. "
+                                 f"Your compiler is unknown. Assuming it supports C++{self._minimum_cpp_standard}.")
+            elif Version(str(self.settings.compiler.version)) < minimum_version:
+                raise ConanInvalidConfiguration(f"{self.ref} requires C++{self._minimum_cpp_standard}, "
+                                                f"which your compiler does not support.")
+            if self.settings.compiler == "clang" and str(self.settings.compiler.version) in ("13", "14"):
+                raise ConanInvalidConfiguration(
+                    f"{self.ref} currently does not work with Clang {self.settings.compiler.version} on CCI, it enters "
+                    f"in an infinite build loop (smells like a compiler bug). Contributions are welcomed!")
 
     def validate(self):
         if self.settings.compiler.get_safe("cppstd"):
-            tools.check_min_cppstd(self, '17')
-
-        if self.settings.compiler == "gcc" and Version(self.settings.compiler.version) < "9":
-            raise ConanInvalidConfiguration("Need GCC >= 9")
-
-        if self.settings.compiler == "clang" and Version(self.settings.compiler.version) < "8":
-            raise ConanInvalidConfiguration("Need Clang >= 8")
-
-        if self.settings.compiler == "Visual Studio" and Version(self.settings.compiler.version) < "15.8":
-            raise ConanInvalidConfiguration("Need Visual Studio >= 2017 15.8 (MSVC 19.15)")
-
-        # Actually, we want *at least* 15.8 (MSVC 19.15), but we cannot check this for now with Conan.
-        if self.settings.compiler == "msvc" and Version(self.settings.compiler.version) < "19.15":
-            raise ConanInvalidConfiguration("Need msvc >= 19.15")
+            check_min_cppstd(self, self._minimum_cpp_standard)
 
+        self._validate_min_compiler_version()
         self._validate_task_system()
+        self._validate_thread_system()
         self._validate_boost_components()
 
     def configure(self):
-        if self.options.task_system == "auto":
-            self.options.task_system = self._default_task_system()
-        self.output.info("Stlab Task System: {}.".format(self.options.task_system))
+        self.output.info("STLab With Boost: {}.".format(self.options.with_boost))
+        self.output.info("STLab Future Coroutines: {}.".format(self.options.future_coroutines))
+        self.output.info("STLab No Standard Coroutines: {}.".format(self.options.no_std_coroutines))
+        self.output.info("STLab Task System: {}.".format(self.options.task_system))
+        self.output.info("STLab Thread System: {}.".format(self.options.thread_system))
```

Comment:
Would it be possible to remove these messages? It's quite weird to see these messages under `======== Computing dependency graph ========` in conan v2, and from my point of view it's the job of upstream build system to display these informations at build time.

Reply:
Need to be done in another PR. This one is already merged